pom.xml

```diff
@@ -33,25 +33,7 @@
 			<groupId>org.springframework.boot</groupId>
 			<artifactId>spring-boot-starter-test</artifactId>		
 			<scope>test</scope>
-		</dependency>		
```

Comment:
Good catch finding that jUnit was included in the other package. Mockito is also included?

Reply:
Correct, [the docs](https://docs.spring.io/spring-boot/docs/current/reference/htmlsingle/#using.build-systems.starters) then `spring-boot-starter-test`.